packages/client/hmi-client/src/page/project/tera-project.vue

```diff
@@ -267,7 +273,7 @@ const props = defineProps<{
 	project: IProject;
 	assetName?: string;
 	assetId?: string;
-	assetType?: ProjectAssetTypes | 'overview' | '';
+	assetType?: ProjectAssetTypes | 'overview' | 'workflow' | '';
```

Comment:
to fix the typecheck issue add 'workflow' to the `Tab` type in 'common.ts`

Reply:
yepp just saw that now, why is this defined in individually in the project page as well as the common.ts?

Reply:
Yeah it's kinda weird. Ideally it should just be `ProjectAssetTypes` so its just one type that can be easily shared but then more tab types came around that weren't necessarily assets. Perhaps a TabType could be made to hold ProjectAssetTypes + other tabs like overview or ''. Doubt it's important to worry about now though